### PR TITLE
Revert node-labeller

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
Revert node-labeller from release-0.29

Since the node-labeller was only partially integrated in release-0.29, the full feature set is only available if the standalone node-labeller is deployed via its operator. This operator conflicts with the virt-operator. To avoid this conflict, we revert the node-labeller parts for release-0.29. This allows using the standalone-labeller in release-0.29.

**Release note**:
```
Remove node-labeller from kubevirt
```

/cc @rmohr, @tiraboschi

### CODE REVIEW

**Comment by rmohr** [comment]
/lgtm
/approve
/hold

Need to enable CI for the branch.

**Comment by rmohr** [comment]
/hold cancel

Tests are in place.

File: pkg/virt-operator/creation/rbac/handler.go [hunk 1]
```diff
@@ -99,22 +99,6 @@ func newHandlerClusterRole() *rbacv1.ClusterRole {
...
-					"watch",
-					"get",
-				},
-			},
-			{
-				APIGroups: []string{
-					"",
-				},
-				Resources: []string{
-					"configmaps",
```
Comment (rmohr) [inline pkg/virt-operator/creation/rbac/handler.go:112]: You will have to keep all configmap logic in the  operator to allow people upgrading. Just remove the configmap.

Comment (ksimon1) [reply pkg/virt-operator/creation/rbac/handler.go:112]: Updated

File: cmd/virt-handler/virt-handler.go [hunk 2]
```diff
@@ -180,19 +180,6 @@ func (app *virtHandlerApp) Run() {
 		cache.Indexers{},
 	)
 
-	nodeLabellerConfigMapSharedInformer := cache.NewSharedIndexInformer(
```
Comment (rmohr) [inline cmd/virt-handler/virt-handler.go:183]: We will have to keep this informer. Otherwise we can't react to it properly (for instance deletes).

**Pull request merged**